purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -353,29 +353,29 @@ class Purchases internal constructor(
     }
 
     /**
-     * Gets the StoreProduct for the given list of subscription skus.
-     * @param [skus] List of skus
+     * Gets the StoreProduct for the given list of subscription and non-subscription productIds.
```

Comment:
nit: i think we could just get rid of "subscription and non-subscription" here. i might think i needed to pass both in separately or something

Reply:
another nit, and not your code but `StoreProduct -> StoreProduct(s)`